flashinfer/gemm/gemm_base.py

```
@@ -7136,6 +7137,623 @@ def group_gemm_fp8_nt_groupwise(
     return out
 
 
+@functools.cache
+def get_gemm_sm120_module_cute_mxfp8():
+    """MXFP8 group GEMM module accessor for SM120."""
+    return gen_gemm_sm120_module_cute_mxfp8().build_and_load()
+
+
+@flashinfer_api
+def mxfp8_quantize_for_zero_padding(
+    input: torch.Tensor,
```

Ruff (0.15.15)

[error] Function argument `input` is shadowing a Python builtin

(A002)